src/Pagination.jsx

```diff
@@ -233,6 +234,7 @@ class Pagination extends React.Component {
             title={props.showTitle ? locale.next_page : null}
             onClick={this._next}
             className={`${this._hasNext() ? '' : `${prefixCls}-disabled`} ${prefixCls}-next`}
+            aria-disabled={!this._hasNext()}
```

Comment:
i will do a little change，to make sure _hasNext will only be called 1 time during every render